src/test/regress/expected/multi_connection_cache.out

```diff
@@ -104,3 +108,11 @@ SELECT get_and_purge_connection('localhost', :worker_port);
 (1 row)
 
 SET client_min_messages TO DEFAULT;
+\c
+-- purge existing connection to localhost
+SELECT connect_and_purge_connection('localhost', :worker_port);
```

Comment:
That's what the /c is there for, right? It reconnects, resetting the
session.

It definitely segfaults (tested it).

--
*Jason Petersen*
Software Engineer | Citus Data
303.736.9255
jason@citusdata.com

On Jul 28, 2016, at 5:53 PM, Andres Freund <notifications@github.com> wrote:

In src/test/regress/expected/multi_connection_cache.out
<https://github.com/citusdata/citus/pull/675#discussion_r72723448>:

> @@ -104,3 +108,11 @@ SELECT get_and_purge_connection('localhost', :worker_port);
>  (1 row)
>
>  SET client_min_messages TO DEFAULT;
> +\c
> +-- purge existing connection to localhost
> +SELECT connect_and_purge_connection('localhost', :worker_port);

I think doing this test last doesn't actually reproduce the issue at hand
here, does it? Because the hash is already created...

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/citusdata/citus/pull/675/files/c52365b51155347a6c01b2179b3ca2b701da0168#r72723448>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AADF8dAjF38sgtiKRM9tSEMoIG9qECWFks5qaUDxgaJpZM4JVeTE>
.